beta-src/src/components/WDInfoDisplay.tsx

```diff
@@ -0,0 +1,148 @@
+import * as React from "react";
+import {
+  Table,
+  TableBody,
+  TableCell,
+  TableHead,
+  TableRow,
+  TableContainer,
+} from "@mui/material";
+// import { makeStyles } from "@mui/core/styles";
+
+// const useStyles: any = makeStyles({
+//   table: {
+//     maxWidth: "250px",
+//   },
+//   tableHeadCell: {
+//   fontWeight: 600,
+//   p: "2px 5px 5px 5px",
+//   border: "none",
+//   },
+//   tableBodyCell: {
+//     p: "0px 5px 0px 5px",
+//     fontSize: "0.7rem",
+//     border: "none",
+//   },
+// });
+
+/**
+ * (optional)If we can import makeStyles from MUI, then we can define an useStyle object to apply styles on repeated tablecell component
+ */
+
+interface WDInfoDisplayProps {
+  title: string;
+  gameTime: string;
+  phase: string;
+  season: string;
+  year: string;
+  gameType: string;
+  playType: string;
+  rank: string;
+  /**
+   * game setting datas which would be passed to the component by parent component/ context/redux store
+   */
```

Comment:
Is this describing `interface WDInfoDisplayProps`.  If so, it should go above line 32.